### ISSUE TITLE
ru docs [improvement]

### ISSUE THREAD

**Issue by 4iloveg** [phalcon/docs#1004]
![image](https://cloud.githubusercontent.com/assets/19300333/26491846/e081c07a-4219-11e7-9282-2640ed184117.png)
This block (and another) does not show any links.
Can be added to styles 
``
a {
    text-decoration: underline;
}
``

**Comment by sergeyklay** [comment]
Cc: @niden

**Comment by niden** [comment]
Which file is this one for?

**Comment by 4iloveg** [comment]
I saw this on the page https://docs.phalconphp.com/ru/latest/index.html

**Comment by niden** [comment]
I spoke to @sergeyklay because I don't understand the message :). We will put this on hold for now and address it once our new documentation is in place. We will then add a message like that to show where volunteers come from etc.

**Comment by niden** [comment]
Closing this - no longer necessary after the new documentation was deployed
